delta/plugins/elasticsearch/src/main/resources/defaults/default-mapping.json

```diff
@@ -56,6 +56,10 @@
       "type": "boolean",
       "copy_to": "_all_fields"
     },
+    "_markedAsDeprecated": {
+      "type": "boolean",
+      "copy_to": "_all_fields"
```

Comment:
It does not provide much value to copy a boolean value in the `_all_fields`

Reply:
Probably. This is the case for the other fields too. We can address the `_all_fields` thing and fix it in a separate PR for all the metadata, not just `_markedForDeletion`